app/views/plans/_share_form.html.erb

```diff
@@ -88,42 +88,51 @@
   <% new_role = Role.new %>
   <% new_role.plan = @plan %>
   <%= form_for new_role, url: {controller: :roles, action: :create }, html: {method: :post} do |f| %>
-    <div class="form-group col-xs-8">
+    <div class="form-group col-xs-6">
       <%= f.hidden_field :plan_id %>
       <%= f.fields_for :user do |user| %>
-        <%= user.label :email, _('Email'), class: 'control-label' %>
-        <%= user.email_field :email, for: :user, name: "user", class: "form-control", "aria-required": true %>
+        <%= user.label :email, _('Email'), class: 'control-label'%>
+        <%= user.email_field :email, for: :user, name: "user", class: "form-control",
+                            "aria-required": true,
+                            'data-toggle': 'tooltip',
+                            'data-html': true,
+                            title: _('Enter the email address of your collaborator: <ul><li> If they are already using DMPonline, they will see this plan on their dashboard, and recieve an email. </li> <li> If they are not currently using DMPonline, they will recieve an email inviting them to the tool so they can collaborate on your plan.</li></ul>') %>
```

Comment:
Use the app name defined in the branding.yml here instead of 'DMPonline'.

Reply:
Good catch,Good catch, completely slipped my mind, 
I'll update